rest-api/api/dv_order_api.py

```diff
@@ -133,8 +144,9 @@ def _put_supply_delivery(self, resource, bo_id):
     order.shipmentEstArrival = eta.date()
     order.shipmentStatus = tracking_status_enum
 
-    self._do_update(order)
-    return self._make_response(order)
+    # self._do_update(order)
+    # return self._make_response(order)
+    return order
```

Comment:
Will this change skip writing the updates?

Reply:
no it is called in updateable api, but I do need to remove the comments